recipes/faampy/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "faampy" %}
+{% set version = "v0.1.4" %}
+
+package:
+    name: {{ name|lower }}
```

Comment:
You define the variable `name` but you use it only here. I'd rather have 1-less line and just write the name here instead.
(Package names don't change that much to deserve its own variable. The `lower` trick is only for an idiosyncrasy on PyPI that allow for case sensitive in the URL but than treat the all package as lower case.)